pandas/conftest.py

```diff
@@ -1692,6 +1693,22 @@ def any_skipna_inferred_dtype(request):
     return inferred_dtype, values
 
 
+# ----------------------------------------------------------------
+# Ctypes
+# ----------------------------------------------------------------
+@pytest.fixture
+def mock_ctypes(monkeypatch):
```

Comment:
Could you just define this fixture in `test_clipboard.py` since it's only used there currently. We can back it out to `pandas/conftest.py` later if more tests in other directories can share this.

Reply:
I had a test in `test_errors.py` that used it, but that one does feel redundant. I remove that one and move this back to `test_clipboard.py`